release.sh

```diff
@@ -79,16 +141,21 @@ function build_release() {
     env CGO_ENABLED=0 GOOS=$os GOARCH=$arch GOARM=$arm go build -v -trimpath -ldflags="${ldflags}" -tags="${buildtags}" ${TAP_PKG}/cmd/tapcli
     popd
 
+    # Add the hashes for the individual binaries as well for easy verification
+    # of a single installed binary.
+    shasum -a 256 "${dir}/"* >> "manifest-$tag.txt" 
+
     if [[ $os == "windows" ]]; then
-      zip -r "${dir}.zip" "${dir}"
+      reproducible_zip "${dir}"
     else
-      tar -cvzf "${dir}.tar.gz" "${dir}"
+      reproducible_tar_gzip "${dir}"
     fi
-
-    rm -r "${dir}"
   done
 
-  shasum -a 256 * >manifest-$tag.txt
+  # Add the hash of the packages too, then sort by the second column (name).
+  shasum -a 256 lightning-terminal-* vendor* >> "manifest-$tag.txt"
+  LC_ALL=C sort -k2 -o "manifest-$tag.txt" "manifest-$tag.txt"
+  cat "manifest-$tag.txt"
```

Comment:
Let me know if you think this is unnecessary for `litd`, but I figured we'd probably want to mimic the `lnd` behaviour as much as possible to be as coherent between the repos as possible.